lxd/resources/storage.go

```diff
@@ -84,8 +84,17 @@ func storageAddDriveInfo(devicePath string, disk *api.ResourcesStorageDisk) erro
 			disk.Serial = udevProperties["E:ID_SERIAL_SHORT"]
 		}
 
-		// Model number
-		if udevProperties["E:ID_MODEL"] != "" {
+		// Model number. Check the encoded model first to match what lsblk does.
+		if udevProperties["E:ID_MODEL_ENC"] != "" {
+			// The encoded model may contain escaped hex values which need to
+			// be converted to ASCII
+			model, err := strconv.Unquote(`"` + udevProperties["E:ID_MODEL_ENC"] + `"`)
```

Comment:
We only use ` for multi-line quoting and tend to prefer Sprintf over string concatenation.

Reply:
Right, we can't use this logic to decode _ENC, there is a lot more to _ENC that this.